BinaryObjectScanner/Protection/SecuROM.cs

```diff
@@ -25,7 +110,11 @@ public class SecuROM : IExecutableCheck<PortableExecutable>, IPathCheck
 
             if (exe.MatroschkaPackage != null)
             {
-                var matroschka = CheckMatroschkaPackage(exe, includeDebug);
+                var package = exe.MatroschkaPackage;
+                if (package == null)
+                    return null;
+                
+                var matroschka = CheckMatroschkaPackage(package, includeDebug);
```

Comment:
Nitpick: Maybe something like `packageVersion` would be appropriate for the return variable name? Not sure why but seeing it just called `matroschka` feels weird.

Reply:
Changed it to packageType